protocol/protocol.go

```diff
@@ -602,7 +604,9 @@ func (o *Operation) MarshalBinary() ([]byte, error) {
 		b = append(b, tlvBytes(TagReqContext, o.ReqContext)...)
 	}
 
-	b = append(b, tlvBytes(TagComplianceRegion, []byte{byte(o.ComplianceRegion)})...)
+	if o.ComplianceRegion != 0 {
```

Comment:
When we are returning packets, the Op is built here -https://github.com/cloudflare/gokeyless/blob/master/protocol/conn.go#L41 - it will be the 0 value for op 100% of the time. With the current implementation that tag would've never been accurate for FRH. There doesn't seem to be value in returning the tlv back to the consumer anyway.

Reply:
We should at least add a comment here that including compliance region is a breaking change for at least 1 client. It's a little fragile as is. Maybe we add a test as well that will fail when we return compliance region.